.coveragerc

```diff
@@ -0,0 +1,11 @@
+# .coveragerc to control coverage.py
```

Comment:
This can be done through the poetry toml config, rather than needing a new file, eg:
https://github.com/python-poetry/poetry/commit/1977945745aeced3376901d4a86b63cbf73e5ed1